### ISSUE TITLE
New 'celltype' suggestions

### ISSUE THREAD

**Issue by archgogo** [HumanBrainProject/openMINDS_controlledTerms#297]
<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/archanag/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/archanag/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link=blue vlink=blue>

name | id | exists as CT? | suggestedTerminology | interlex ID | Notes | definition
-- | -- | -- | -- | -- | -- | --
cortical neuron | https://editor.kg.ebrains.eu/instances/48987687-334a-4ed9-9fba-45e4d0ccf6ad | no | cell type |   |   |  
hippocampal neuron | https://editor.kg.ebrains.eu/instances/222359f6-effa-47bc-874b-dbeceb51f232 | no | cellType |   |   | Aneuron located in the hippocampus
layer 6 neuron | https://editor.kg.ebrains.eu/instances/e04fa358-ecb3-4dce-8624-0aa2f6b62b87 | no | cellType |   | There is cortical layer 6 in UBERONParcellation | neurons in the cortical 6 layer
zinc containing neurons and glia | https://editor.kg.ebrains.eu/instances/8540a1c4-4c66-4be0-be58-55816bd113af | no | cellType (can be split into two) |   |   | Neurons and glia containing zinc. The zinc containing neurons   are a subtype of glutamatergic neurons

</body>

</html>

**Comment by lzehl** [comment]
@archgogo thank you for these suggestions. @UlrikeS91 , @tgbugs this issue should be reviewed in respect to the discussion in #149 and #257

**Comment by lzehl** [comment]
@archgogo thank you for being patient with this. Please use for cortical neuron, hippocampal neuron, layer 6 neuron please the combination of  'neuron', in addition with the anatomical region/entity you need (e.g., 'neuron' and 'hippocampus)

TODO: add 'zinc releasing neuron'

**Comment by archgogo** [comment]
@lzehl, couldn't find any references for 'zinc containing neuron' or 'zinc releasing neuron' on ontology sites. Should I still add it in the 'cellType'

**Comment by lzehl** [comment]
@archgogo yes. Just leave the ontology links empty (null). They are on it's way ;) Nonetheless please provide a definition
